docs/source/guides/using_dask_entitysets.ipynb

```diff
@@ -194,17 +194,17 @@
     "\n",
-    "By default, when Woodwork checks that pandas DataFrames have unique index values. Because performing this same check with Dask would require an expensive compute operation, this check is not performed when adding a Dask DataFrame to an `EntitySet`. When using Dask DataFrames, users must ensure that the supplied index values are unique.\n",
+    "By default, Woodwork checks that pandas DataFrames have unique index values. Because performing this same check with Dask would require an expensive compute operation, this check is not performed when adding a Dask DataFrame to an `EntitySet`. When using Dask DataFrames, users must ensure that the supplied index values are unique.\n",
     "\n",
     "When using a pandas DataFrames, the ordering of the underlying DataFrame rows is maintained by Featuretools. For a Dask DataFrame, the ordering of the DataFrame rows is not guaranteed, and Featuretools does not attempt to maintain row order. If ordering is important, close attention must be paid to any output to avoid issues.\n",
     "\n",
     "### EntitySet Limitations\n",
     "\n",
     "When creating a Featuretools `EntitySet` that will be made of Dask DataFrames, all of the DataFrames used to create the `EntitySet` must be of the same type, either all Dask DataFrames or all pandas DataFrames. Featuretools does not support creating an `EntitySet` containing a mix of Dask and pandas DataFrames.\n",
     "\n",
-    "Additionally, the `EntitySet.add_interesting_values()` method is not supported when using a Dask `EntitySet`. If needed, users can manually set `interesting_values` for DataFrames by assigning them directly with syntax similar to this: `es[\"dataframe_name\"].ww[\"variable_name\"].ww.metadata[\"interesting_values\"] = [\"Value 1\", \"Value 2\"]`.\n",
+    "Additionally, the `EntitySet.add_interesting_values()` method is not supported when using a Dask `EntitySet`. If needed, users can manually set `interesting_values` for DataFrames by assigning them directly with syntax similar to this: `es[\"dataframe_name\"].ww.columns[\"column_name\"].metadata[\"interesting_values\"] = [\"Value 1\", \"Value 2\"]`.\n",
```

Comment:
I believe the original suggestion here `es["dataframe_name"].ww["column_name"].ww.metadata["interesting_values"] = ["Value 1", "Value 2"]` will fall into the trap of setting the metadata on the separate Series object created by `es["dataframe_name"].ww["column_name"]`. 

Switched to going through the `columns` attr to avoid that. @rwedge had brought this gotcha up in the feature descriptions guide as an important thing to warn users away from doing. I'm thinking it might make sense to add that warning here and in the koalas guide as well.

Reply:
Maybe the better approach is to demonstrate using the `add_interesting_values` function directly, similar to what we do in the test below?
```python
def test_add_interesting_values_specified_vals(es):
    product_vals = ['coke zero', 'taco clock']
    country_vals = ['AL', 'US']
    interesting_values = {
        'product_id': product_vals,
        'countrycode': country_vals,
    }
    es.add_interesting_values(dataframe_name='log', values=interesting_values)
```

I guess I had forgotten that we reworked this to allow for this type of usage. This approach should work fine with Dask/Koalas entitysets.

Reply:
oh yeah, forgot about that as well. Definitely a nicer way to set these values; I'll update both guides to show an example

Reply:
Updated here and in the koalas guide to show an example with the call to `add_interesting_values`